vars/vegautils.groovy

```diff
@@ -111,4 +111,12 @@ String getVegaCiBotCredentials() {
   Collections.shuffle credentialNames
 
   return credentialNames.first()
-}
\ No newline at end of file
+}
+
+def dockerCleanup() {
```

Comment:
I would add universal Cleanup and call it agentCleanup but it is fine as it is now :)